pkg/xen-tools/initrd/udhcpc_script.sh

```diff
@@ -67,6 +69,16 @@ install_classless_routes()
     done
 }
 
+update_resolv_conf() {
```

Comment:
Shouldn't this function take arguments ($1 through $n) instead of referencing globals?

Reply:
Fair point, done.